mergify_engine/context.py

```diff
@@ -63,25 +63,74 @@ class Installation:
     subscription: subscription.Subscription
     client: github.GithubInstallationClient
 
+    repositories: "typing.Dict[github_types.GitHubRepositoryName, Repository]" = (
+        dataclasses.field(default_factory=dict)
+    )
```

Comment:
Do you really need to store them?

Reply:
This is planned to be replace this Context construction here: https://github.com/Mergifyio/mergify-engine/pull/1729/files#diff-e709bf7f7a0ceb20e0cfafd3b8ab1e163a0c04e84262742607dc6828cf868770R258